gst/nnstreamer/elements/gsttensor_trainer.c

```diff
@@ -735,10 +747,12 @@ gst_tensor_trainer_chain (GstPad * sinkpad, GstObject * parent,
   return GST_FLOW_OK;
 
 error:
-  mem_blocks = gst_buffer_n_memory (inbuf);
-  for (i = 0; i < mem_blocks; i++) {
-    if (in_mem[i])
+  num_tensors = gst_buffer_n_memory (inbuf);
```

Comment:
IMO, this line is unnecessary because num of tensors at line 574 would not be changed.

Reply:
Thank you. I will remove this line.